packages/flutter/lib/src/material/page.dart

```diff
@@ -64,9 +124,6 @@ class MaterialPageRoute<T> extends PageRoute<T> {
   Color get barrierColor => null;
 
   @override
-  bool canTransitionFrom(TransitionRoute<dynamic> nextRoute) => false;
```

Comment:
Yes, at least, in the case where it's going to do the cupertino thing. Maybe not on Android?